lib/conduit_lwt_tls.mli

```diff
@@ -32,9 +32,10 @@ module Server : sig
     Tls.Config.server ->
     Lwt_unix.file_descr ->
     (Lwt_unix.file_descr * Lwt_io.input_channel * Lwt_io.output_channel) Lwt.t
+  [@@@deprecated "Low level function. Will be removed."]
```

Comment:
So there seems to be no way to provide a `Tls.Config.server` to a new conduit server. Unless you're writing a TCP server from scratch and use this accept function in your accept loop.

How about adding a new version of init that does provide this functionality? The current init version can be implemented on top of that and eventually removed.

@avsm Will do.